ts-patch.yaml

```
@@ -0,0 +1,66 @@
+package:
+  name: ts-patch
+  version: 3.2.1
+  epoch: 0
+  description: "Patch typescript to allow custom transformers (plugins) during build"
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - nodejs
+      - npm
+      - typescript
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+      - npm
+      - rsync
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nonara/ts-patch.git
+      tag: v${{package.version}}
+      expected-commit: 41b2112495265819558e48f3d56a81dfb4533005
+
+  - name: npm install
+    runs: |
+      npm install && npm run build
+      mkdir -p ${{targets.destdir}}/usr/lib/node_modules/ts-patch
+      rsync -av --exclude='melange-out' /home/build/* ${{targets.destdir}}/usr/lib/node_modules/ts-patch
+
+      mkdir -p ${{targets.destdir}}/usr/bin/
+      ln -sf /usr/lib/node_modules/ts-patch/node_modules/ts-patch/bin/tspc.js ${{targets.destdir}}/usr/bin/tspc
+      ln -sf /usr/lib/node_modules/ts-patch/node_modules/ts-patch/bin/ts-patch.js ${{targets.destdir}}/usr/bin/ts-patch
+
+update:
+  enabled: true
+  github:
+    identifier: nonara/ts-patch
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - nodejs
+        - npm
+        - coreutils
+  pipeline:
+    - name: Test tspc version
+      runs: |
+        tspc --version
+    - name: Test tspc compiles
+      runs: |
+        tsfile=$(mktemp /tmp/tempfile.XXXXXX.ts)
+        echo "console.log('Green eggs and ham.');" > $tsfile
+        jsfile=$(mktemp /tmp/tempfile.XXXXXX.js)
+        tspc $tsfile --outFile $jsfile
+        node $jsfile
```

```suggestion
        node $jsfile |grep "Green eggs and ham."
```

Super small suggestion here - since the application is logging this output, the test can check for it. This ensures that it is actually running the ts application, as opposed to say, silently outputting nothing. Looks great!